frontend/components/modal/FakeUsers.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts">
+const { fakeUsers: { fakeUsersVisible } } = useModal()
+const { t } = useI18n()
+const { getMaybeMe, refreshMaybeMe } = useSession()
+const { isAuthenticated, signIn, signOut } = await useMSAL()
+const localePath = useLocalePath()
+
+const prefix = 'ModalFakeUsers'
+const tt = (s: string) => t(`${prefix}.${s}`)
+
+// TODO(grady) move this to an env-specific location, omit the component if empty.
+const hardcodedLocalUsers = [{
+  id: 'user.c9a959a1c4d6b8be1fc1',
+  email: 'rmi-localtest-a@siliconally.org',
+  password: 'CQdCZd9lq2HOnFB',
+}, {
+  id: 'user.4aed45b19c7ef24e105c',
+  email: 'rmi-localtest-b@siliconally.org',
+  password: 'Dq7CLBX90nFtNk8',
+}]
+
+const { maybeMe } = await getMaybeMe()
+
+const users = computed(() => {
+  const me = maybeMe.value
+  return hardcodedLocalUsers.map((u) => ({
+    itMe: me && u.email.toLowerCase() === me.enteredEmail.toLowerCase(),
```

Comment:
`helo, it me`

but also maybe `isLoggedInUser`?

Reply:
Not sure what you mean here? I don't think we have that utility?

Reply:
Lol I'm just noting that you've named it `"it me"`